packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -470,10 +471,47 @@ export default class Meetings extends WebexPlugin {
      * @param {String} [options.feedbackId] ID used for tracking
      * @param {String} [options.locusId]
      * @param {Stirng} [options.correlationId]
+     * @param {String} [options.meetingId]
      * @returns {String} feedback ID logs were submitted under
      */
     uploadLogs(options = {}) {
-      return this.loggerRequest.uploadLogs(options);
+      if (this.config.autoUploadLogs) {
```

Comment:
The `meetings.uploadLogs` feature allows developers to upload the logs whenever they like. Adding this config check here will require the auto upload log config to be enabled even if the developer calls `meetings.uploadLogs` themselves.

Reply:
correct changed it